Civi/RemoteTools/Api4/Api4.php

```diff
@@ -40,12 +40,11 @@ public function __construct() {
     self::$instance = $this;
   }
 
-  public function countEntities(string $entityName, ConditionInterface $where, array $options = []): int {
+  public function countEntities(string $entityName, ConditionInterface $where, array $extraParams = []): int {
     return $this->execute($entityName, 'get', [
-      'checkPermissions' => $options['checkPermissions'] ?? FALSE,
```

Comment:
Assuming and documenting that `checkPermissions` defaults to `FALSE` if not passed to `execute()`, right?

Reply:
`checkPermissions` defaults to FALSE as before and is documented here: https://github.com/systopia/de.systopia.remotetools/blob/0855e8c86791c2282478747ce3af659949438761/Civi/RemoteTools/Api4/Api4Interface.php#L30